README.md

```diff
@@ -27,23 +33,17 @@
 ### Setup
 
 1. Open a command line
-2. Clone the project to your computer with `git clone git@github.com:ShadyShawkat/React-Group-Project.git`
-3. Go into the folder to the project with `cd React-Group-Project`
+2. Clone the project to your computer with `git clone git@github.com:ShadyShawkat/World-covid-tracker.git`
+3. Go into the folder to the project with `cd World-covid-tracker`
 4. Build the node modules with `npm install`. 
+5. Run the Project using `npm start`
 
 ### Usage
 
 Any change to the project should be done in the folder `src`.
 
 ## Author
 
-👤 **Will Teixeira**
-
-- GitHub: [@githubhandle](https://github.com/iwillteixeira)
-- Twitter: [@twitterhandle](https://twitter.com/iwillteixeira)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juscelinodev/)
-
-
 👤 **Shady Shawkat**
 
```

Comment:
- [ ] It would be better if you give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

Kindly add this 😄 👍